src/dashboard/src/templates/ingest/grid.html

```diff
@@ -71,32 +62,6 @@
 
 {% block content %}
 
-  {% if search_transfers_enabled %}
-    {% include "ingest/backlog/_search_form.html" with show_files=False %}
-    <div class="activity-indicator">
-      <img src='/media/images/ajax-loader.gif' />
-    </div>
-
-    <div style='float:left;'>
-      <div id='originals' class='backbone-file-explorer' style='float:left; width: 460px; margin-bottom: 10px;'></div>
-      <div id='originals_controls'>
-        <span id='open_originals_file_button' class='btn btn-default'>{% trans "View file" %}</span>
-        <span id='originals_hide_button' class='btn btn-default'>{% trans "Hide" %}</span>
-      </div>
-    </div>
-    <div style='margin-left: 30px; float:left;'>
-      <div id='arrange' class='backbone-file-explorer' style='float:left; width: 460px; margin-bottom: 10px;'></div>
-      <div id='arrange_controls'>
-        <span id='arrange_create_directory_button' class='btn btn-default'>{% trans "Add directory" %}</span>
-        <span id='arrange_edit_metadata_button' class='btn btn-default'>{% trans "Edit metadata" %}</span>
-        <span id='arrange_delete_button' class='btn btn-default'>{% trans "Delete" %}</span>
-        <span id='arrange_create_sip_button' class='btn btn-primary'>{% trans "Create SIP" %}</span>
-      </div>
-    </div>
-    <br clear='all' />
-    <br />
-  {% endif %}
-
```

Comment:
From a usability perspective, considering that the old SIP arrangement widget is being removed, I wonder if we should have some kind of link at the top of the Ingest grid linking users to Appraisal? I'm not really a user but I'd definitely miss it since I didn't really use Appraisal myself.

Probably too simple but something like the following could be helpful:

---

![image](https://user-images.githubusercontent.com/606459/54753854-4f126680-4b9f-11e9-9f71-c6a94606a553.png)

---

CC @sallain @sromkey @ablwr @ross-spencer @peterVG @evelynPM @currmie @corinnemrogers @kellyannewithane - thoughts?